run_sorting_hat_test_data.sh

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env bash
 
 GTF=$( pwd )/test-data/gff
-DELETE_PREV_OUTPUT=False
+FORCEALL=$true
+RESTART_TIMES=1
 SKIP_STEPS=$( pwd )/step_skip.yaml
+CHECK_SPECIES=$( pwd )/Bulk_recalculations_check_species_names.yaml
```

Comment:
A better name for this file would be `atlas-species-name-mapping.yaml`, since it won't be used only on recalculations.

Reply:
See c44db20